runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,20 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version; class=%2$.*1$, offset=%6$u, major.minor=%3$u.%4$u, maximum major value=%5$u.0
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.explanation=The major version in the classfile is unsupported on the VM rejecting the classfile.
+J9NLS_CFR_ERR_MAJOR_VERSION2.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MAJOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+
+# END NON-TRANSLATABLE
+
+
+J9NLS_CFR_ERR_MINOR_VERSION2=bad minor verion of class %2$.*1$s. Got minor verion .%4$u, but .0 is the only valid minor version for the major version %3$u; offset=%5$u
```

Comment:
Where did the J9NLS_CFR_ERR_MINOR_VERSION2 message come from, was it made up or copied from Hotspot?

Reply:
verion is spelled wrong

Reply:
It was made up by myself.